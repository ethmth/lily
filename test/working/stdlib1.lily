
let a::int = [3, 2, 1]

# printl prints a list
printl(a)

let b:: bool = [true, false]

for value in b:
    # printb prints a boolean
    printb(value)
