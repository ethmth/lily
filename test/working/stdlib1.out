[3, 2, 1]
true 
false 
